recipes/org-appointments-sidecar

```diff
@@ -0,0 +1,8 @@
+; SPDX-FileCopyrightText: 2024 Samuel W. Flint <me@samuelwflint.com>
+;
+; SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
This is unusual and not something we typically have in the recipes.  I think it's better to just remove it and keep license information on your repo.

Reply:
Ha, whoops!  Sorry about that.  Fixed.